### DISCUSSION TITLE
Error in my code or linguist?

### DISCUSSION THREAD

**Discussion by Coder2195** [github-linguist/linguist#5433] [Classification]
https://github.com/Coder2195Text/space-chaos

Using gitattributes to remove false reporting node_modules and source, I only want the release to be reported.

However, after trying and failing I couldn't get them to hide node_modules and source in linguist-vendored, whats wrong with my code and/or linguist?

**Comment by lildude** [comment]
You've done nothing wrong and you don't even need an override for node_modules as Linguist ignores node_modules by default. The only files being counted are those outside of that directory. The files in source won't be counted because of your override.

It's worth pointing out that Linguist has no control over Search which is what is still showing your files when you click JavaScript in the sidebar. If it did, you wouldn't be able to find any files Linguist ignores.

As this isn't a bug, I'm going to move this to discussions.

**Reply by Coder2195** [reply]
but... it doesnt work, any way to fix it?

**Reply by Coder2195** [reply]
it is falsy reporting and look at my stats
![image](https://user-images.githubusercontent.com/63069499/122654226-faf8e280-d117-11eb-994f-8896659cf5dd.png)
It's all wrong

**Reply by lildude** [reply]
I think this might be a caching issue with the front end servers as I see this:

![2107998E-3397-44FD-8FF4-BBE6CC79FA76](https://user-images.githubusercontent.com/627280/122661243-097ae480-d180-11eb-9b18-6faf1b91df6f.jpeg)

**Reply by lildude** [reply]
Wait a moment, where are you seeing that? The repo you referenced originally doesn't have any Python code.
